lib/components/DynamicListTile.dart

```diff
@@ -76,7 +106,13 @@ List<Fields> fields = <Fields>[
   ),
   Fields(
     'SELECT COLOR',
-    <String>['BLUE', 'RED', 'YELLOW', 'GREEN', 'PURPLE'],
```

Comment:
Why not add another field to the Fields class called `color` and instead of having to check for the color name you're just iterating through the `color` attribute type now